svc/ChangelogService.js

```diff
@@ -61,7 +61,7 @@ export class ChangelogService extends HoistService {
 
     /** @return {?string} */
     get latestAvailableVersion() {
-        if (!this.enabled) return null;
+        if (!this.enabled || includes(this.versions[0].version, 'SNAPSHOT')) return null;
```

Comment:
So I do find this a bit confusing to have the getter return null in the snapshot case - snapshots are valid versions, and there can be a CL entry for a snapshot, so it's "available" as far as this getter and its area of responsibility are concerned (which are limited to checking that the service is generally enabled and then returning the first version string from the list).

Instead I would put a new conditional within `markLatestAsRead()` since that's the specific behavior we want to adjust.  That method could check if `latestAvailableVersion` includes "SNAPSHOT" and return early - might as well log a slightly different console message in that case since we're bothering to log one in the existing case within that method already.

Since that getter is not used by any other methods at the moment the end functional result is the same, but going forward we might find it helpful to have the getter continue to return a version when enabled, even a snapshot. Hope that rationale for changing 'markLatestAsRead()' vs the more general getter sounds sensible / meaningful all around.

Reply:
That is a much more sensible approach!